### PR TITLE
Feed design

### PR DESCRIPTION
**Pull request by aublhr99**

### CODE REVIEW

**Review by jennhe — CHANGES_REQUESTED** [review]
details view does not show up

File: app/src/main/java/com/example/teamproject/RecAdAdapter.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RecAdAdapter extends RecyclerView.Adapter<RecAdAdapter.MasonryView> {
+    private static List<Ad> mAds;
+    private Context context;
+
+    public int getAdapterPosition() {
+        return getAdapterPosition();
```
Comment (baldwinchang) [inline app/src/main/java/com/example/teamproject/RecAdAdapter.java]: Is this infinite recursion?

Comment (aublhr99) [reply app/src/main/java/com/example/teamproject/RecAdAdapter.java]: fixed

**Review by jennhe — APPROVED** [review]
good work!!!!

**Pull request merged**